Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2327 -- Item counts on subject lookup homepage

Resolves [EREGCSC-2327](https://jiraent.cms.gov/browse/EREGCSC-2327)

**Description**

Each subject in the main subject "Table of Contents" (Subject ToC) area of the page on the "policy repository" homepage (currently /policy-repository/) communicates how many public & internal resource items are associated with that subject, matching the [Figma design](https://www.figma.com/proto/KiWw729Nct6tY9gzmvEkVO/Subject-filter-only?page-id=3%3A19&type=design&node-id=379-102357&viewport=1140%2C302%2C0.38&t=LWK6BE8SKAoeMkam-1&scaling=min-zoom&starting-point-node-id=112%3A4972).

**This pull request changes:**

- No longer caches `file-manager/subjects` API endpoint -- fetches response from the server on every page load
- Displays public and internal document counts underneath each subject in the Subject ToC

**Steps to manually verify this change:**

1. All tests pass
2. Visit [experimental deployment](https://q9sjowh0c0.execute-api.us-east-1.amazonaws.com/dev1022) and see that the number of documents match what you expect
3. Upload an internal document and then refresh the page and make sure the number of internal documents for that subject increases as expected
